
class ChainException(Exception):
    pass


class PeerNotFoundException(ChainException):
    pass
